src/assets/locales/en/translation.json

```diff
@@ -28,5 +28,9 @@
     "ETH Balance": "ETH Balance",
     "Members": "Members",
     "Plugin activation time": "The plugin will be activated at",
+    "3BoxLoginSuccess": "Logged in to 3Box",
+    "3BoxProfileSuccess": "Profile data saved to 3Box",
+    "Following": "Now following",
+    "UnFollowing": "No longer following"
```

Comment:
`'` missing

Reply:
Fixed.  Dealing with constant merge conflicts in this file is becoming a pain.  Wondering if we can figure out a way of editing this file that will minimize that.